cumulus-model/src/main/java/mil/army/usace/hec/cumulus/client/model/CumulusObjectMapper.java

```diff
@@ -39,6 +39,10 @@ public static String mapObjectToJson(Object object) throws IOException {
         return OBJECT_MAPPER.writeValueAsString(object);
     }
 
+    public static String getValueForKey(String json, String key) throws IOException {
+        return OBJECT_MAPPER.readTree(json).get(key).asText();
```

Comment:
won't this throw a null pointer exception if the node doesn't exist? given this is public API, you'd want to protect against invalid input

Reply:
good point. Updated to throw IOException if key doesn't exist